.github/workflows/system-tests.yml

```diff
@@ -456,3 +456,29 @@ jobs:
           package-name: 'system-tests/${{ matrix.image }}'
           package-type: 'container'
         continue-on-error: true
+
+  build-artifacts:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+    steps:
+      - run: mkdir binaries/
+      - uses: actions/checkout@v4
+        with:
+          path: binaries/dd-trace-rb/
+      - name: Upload artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: system_tests_binaries
+          path: binaries/
+
+  parametric:
+    needs:
+      - build-artifacts
+    uses: DataDog/system-tests/.github/workflows/run-parametric.yml@main
+    secrets: inherit
+    with:
+      library: ruby
+      binaries_artifact: system_tests_binaries
+      _experimental_job_count: 8
+      _experimental_job_matrix: '[1,2,3,4,5,6,7,8]'
```

Comment:
I'm not entirely sure if these changes should go to master; I've pinged folks about it internally.

Reply:
Yes, we should run parametric tests. This may have caught the regression (or may not, because parametric test CI did pass on system tests repository, but you get the idea about coverage within our CI)